packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -20,33 +20,88 @@ import {
   useImperativeHandle,
 } from "react";
 import {
-  Box,
+  Box as MuiBox,
   Checkbox as MuiCheckbox,
-  Chip,
+  Chip as MuiChip,
   ListItemSecondaryAction,
   ListSubheader,
-  MenuItem,
+  MenuItem as MuiMenuItem,
   Select as MuiSelect,
+  SelectChangeEvent,
 } from "@mui/material";
```

Comment:
We should really make a MUI re-export file that includes all of these renames :p.